### DISCUSSION TITLE
Understanding Yeast-GEM deviation from Imm904 / forster 2003

### DISCUSSION THREAD

**Discussion by Orenhy** [SysBioChalmers/yeast-GEM#359] [HELP!]
Hi Yeast-GEM developers.
Have recently transitions from modeling using iMM904 to yeast8, been having some issues understanding the root of changes made to the biomass function.
The general source for both iMM904 and yeast8 seems to be the work by forster and colleagues in 2003 
Paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC420374/
supplementary: https://genome.cshlp.org/content/suppl/2003/02/03/13.2.244.DC1/3.pdf
iMM904 conforms exactly numbers in paper, while biomass function in yeast8 cites it as source it deviates in many ways, some examples:
* All amino acids besides cysteine are multiplied by 1.14 compared to iMM904/forster
* '1 3 beta D Glucan' is split in yeast8 to also specify  '1 6 beta D Glucan' and but give different values that don't sum to original number
* the biomass yield of yeast8 is lower than imm904, and my attempts to convert and sum mol of components is not adding up to 1 gram.

Is there any table / sheet that can be shared which details exactly what modifications have been made to the yeast8 biomass function, and why? 

Thanks for the time and any information that can be given here!

**Comment by edkerk** [comment]
The biomass composition since Yeast-GEM 8.3.1 has been based on the curation done [here](https://github.com/SysBioChalmers/yeast-GEM/blob/main/code/.deprecated/modelCuration/fixBiomassComp.m), also mentioned in PR #159. Note that if you want to rerun that code, it requires the file and folder structure from [Yeast-GEM 8.3.1](https://github.com/SysBioChalmers/yeast-GEM/releases/tag/v8.3.1).

Briefly, the biomass composition is still based on Forster et al, 2003, but complemented with some more recent data where multiple macromolecules were measured from the same cultivations, and scaled to end up with 1 g/gDCW.

You can run [sumBioMass](https://github.com/SysBioChalmers/yeast-GEM/blob/main/code/otherChanges/sumBioMass.m) (also on the latest yeast-GEM) to calculate that all biomass components add up to 1 gram.
